src/com/android/volley/toolbox/NetworkMonitor.java

```diff
@@ -57,7 +60,7 @@ public static void add(NetworkResponse response, ImageRequest request) {
             if (bandwidthKBPS < BANDWIDTH_HYSTERETIC_LOWER) {
                 lowBandwidth = true;
                 log("NETMON", "Low-Bandwidth flag On");
-            } else if (bandwidthKBPS > BANDWIDTH_HYSTERETIC_UPPER) {
+            } else if (bandwidthKBPS > BANDWIDTH_HYSTERETIC_UPPER && (!mIsRoaming || !mForceLowBandwidthIfRoaming)) {
```

Comment:
i haven't seen a truth table since college. brings me back. 

thank you, sean.